app/assets/stylesheets/application.scss

```diff
@@ -46,3 +46,23 @@ p {
   height: 1px;
   overflow: hidden;
 }
+
+.hide-bullet-points {
+  list-style: none;
+}
+
+.hide-bullet-points li{
+  margin: 5px 0;
+}
+
+.mobile-display-only{
+  color: white !important;
```

Comment:
`!important` tags are bad, why did you want to use it here?

Reply:
Resolved in 91ff03d